prow/tide/tide.go

```diff
@@ -35,13 +37,20 @@ import (
 	"k8s.io/test-infra/prow/pjutil"
 )
 
+const (
+	tideStatusContext   string = "tide"
+	tideStatusInPool           = "In tide pool. Will consider for merge on green."
```

Comment:
`s/on green/???` something more technical/specific here may be appropriate? "On green" leaves out information about e.g. required contexts.

Reply:
I'm planning to put up a PR later adding some enhancements to the status page. Since this is a blanket requirement in tide it seems like we should highlight this clearly once instead of putting it on every entry.

Reply:
Tweaked it a bit.